Source/Utils/Average.H

```diff
@@ -57,38 +52,145 @@ namespace Average{
-     *        <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
-     *        \c mf_in into the output
-     *        <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
-     *        \c mf_out.
+     *        in the input MultiFab \c mf_in into the output MultiFab \c mf_out.
      *
-     * \param[in,out] mf_out <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
-     *                       containing the floating point data to be filled with cell-centered averages
-     * \param[in]     mf_in  <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
-     *                       containing the floating point data to be averaged
-     * \param[in]     dcomp  offset for the fourth component of the
-     *                       <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                       object, extracted from its
-     *                       <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
-     *                       , where the cell-centered averages will be stored
-     * \param[in]     scomp  optional offset for the fourth component of the
-     *                       <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                       object, extracted from its
-     *                       <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
-     *                       , containing the data to be averaged
+     * \param[in,out] mf_out MultiFab containing the floating point data to be filled
+     *                       with cell-centered averages
+     * \param[in]     mf_in  MultiFab containing the floating point data to be averaged
+     * \param[in]     dcomp  offset for the fourth component of the Array4 object,
+     *                       extracted from its MultiFab, where the cell-centered
+     *                       averages will be stored
+     * \param[in]     scomp  optional offset for the fourth component of the Array4
+     *                       object, extracted from its MultiFab, containing the data
+     *                       to be averaged
      * \param[in]     ncomp  optional number of components to loop over for both the
-     *                       input and the output
-     *                       <a href="https://amrex-codes.github.io/amrex/doxygen/structamrex_1_1Array4.html">Array4</a>
-     *                       objects, extracted from their respective
-     *                       <a href="https://amrex-codes.github.io/amrex/doxygen/classamrex_1_1MultiFab.html">MultiFab</a>
+     *                       input and the output Array4 objects, extracted from their
+     *                       respective MultiFab
      */
```

Comment:
Reason for cleaning up this Doxygen lines: the web links I added in an earlier PR made the comments non-readable in the code itself, hence I removed them. These changes are not strictly related to this PR, but most of this file is being modified anyways and I'm taking the opportunity to make the Doxygen documentation uniform among the functions defined in this whole new namespace.